build.sbt

```diff
@@ -171,8 +171,8 @@ lazy val thrall = playProject("thrall", 9002)
 
 lazy val usage = playProject("usage", 9009).settings(
   libraryDependencies ++= Seq(
-    "com.gu" %% "content-api-client-default" % "19.0.4",
-    "com.gu" %% "content-api-client-aws" % "0.7",
+    "com.gu" %% "content-api-client-default" % "32.0.0" exclude("org.slf4j","slf4j-api"), // slf4j exclusion can be removed when we upgrade to SLFJ v2
+    "com.gu" %% "content-api-client-aws" % "0.7.6" exclude("org.slf4j","slf4j-api"), // slf4j exclusion can be removed when we upgrade to SLFJ v2
```

Comment:
After https://github.com/guardian/grid/pull/4377 was merged yesterday, the rest of the app is on slf4j v2, so if we rebase this branch then we shouldn't need these exclusions any more 🎉 
Worth giving a quick test after rebasing that the Play upgrade doesn't cause any other annoying conflicts in here too

Reply:
tested in `TEST` after this rebase 👍